Vagrantfile

```diff
@@ -0,0 +1,18 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "centos/7"
+  config.vm.synced_folder '.', '/vagrant', disabled: true
+  if Vagrant::Util::Platform.windows?
+    target_path = ENV['USERPROFILE'].gsub(/\\/,'/').gsub(/[[:alpha:]]{1}:/){|s|'/' + s.downcase.sub(':', '')}
+    config.vm.synced_folder ENV['USERPROFILE'], target_path, type: 'sshfs', sshfs_opts_append: '-o umask=000 -o uid=1000 -o gid=1000'
+  else
+    config.vm.synced_folder ENV['HOME'], ENV['HOME'], type: 'sshfs', sshfs_opts_append: '-o umask=000 -o uid=1000 -o gid=1000'
+  end
+
+  config.vm.provision "shell", inline: <<-SHELL
+    sudo yum install -y @"Development Tools" rpm-build gcc-c++
```

Comment:
Technically not required. I took this from another setup where I need it. I think I remove _rpm-build_ and _gcc-c++_. Might make sense to keep _@Development Tools_, since it contains some tools which are handy to have on the VM, eg git.